lib/pure/collections/lists.nim

```diff
@@ -498,6 +499,20 @@ proc add*[T](L1, L2: var SinglyLinkedList[T]) {.since: (1, 5).} =
   L2.head = nil
   L2.tail = nil
 
+proc add*[T](L1: var SinglyLinkedList[T], L2: SinglyLinkedList[T]) {.since: (1, 5).} =
```

Comment:
let's not propagate pre-existing bad code :)
this violates style guide (caps)

instead use:
```nim
proc add*[T](a: var SinglyLinkedList[T], b: SinglyLinkedList[T]) {.since: (1, 5).} =
```
(or s/a/list/ but it's implied by the type)

future PR can fix the pre-existing style violations